deploy/balena-k3s/bastion/Dockerfile

```diff
@@ -51,10 +51,6 @@ RUN arkade version && \
 RUN mkdir -p /app/edge-endpoint
 COPY . /app/edge-endpoint
 
-# Set environment variables for running cluster setup based on env var EDGE_INFERENCE_FLAVOR, defaults to CPU if not set
```

Comment:
This means you're defaulting to GPU on Balena?

Reply:
Yes, I think since our edge will be mostly running with GPU it is ok if we set it to use GPU by default. If we want to change that it is probably better if we moved this to `cluster_setup.sh` instead.

For now, RPi will need to set env var `INFERENCE_FLAVOR` to `CPU` in the balena dashboard.